### DISCUSSION TITLE
p2g analysis on subset of peaks

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1310] [Questions / Documentation] [answered]
I was wondering if it's possible to perform peak 2 gene analysis only on a subset of peaks?
Thank you!

**Comment by rcorces** [comment, marked as answer]
just subset your peakset and re-add it to your project etc

**Reply by rojinsafavi** [reply]
Thanks @rcorces 

Could you please refer me to the ArchR functions that I should use? how should I subset my peakset, and how can I add that to an ArchR project to replace the current peakset or save it as a new obj?

**Reply by rcorces** [reply]
you can find all of the relevant functions here: https://www.archrproject.com/reference/index.html

`getPeakSet()` gives you a GRanges object of your peak set. Alter that however you want
`addPeakSet()` overwrites your current projects peak set
`addPeakMatrix()` to re-calculate the peak matrix
